scripts/macosx/build_lv2.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Echo commands
+set -x
+
+# Quit on errors
+set -e
+
+# Get the path to our scripts folder.
+pushd `dirname $0` > /dev/null
+PROGDIR=`pwd -P`
+popd > /dev/null
+
+export VERSION_NUMBER=1.14.0
+export VERSION=lv2-${VERSION_NUMBER}
+export ARCHIVE=$VERSION.tar.bz2
+
+echo "Building $VERSION for $MIXXX_ENVIRONMENT_NAME for architectures: ${MIXXX_ARCHS[@]}"
+
+for ARCH in ${MIXXX_ARCHS[@]}
+do
+  mkdir -p $VERSION-$ARCH
+  tar -jxf $DEPENDENCIES/$ARCHIVE -C $VERSION-$ARCH --strip-components 1
+  cd $VERSION-$ARCH
+  source $PROGDIR/environment.sh $ARCH
+
+  ./waf configure build --lv2dir "$MIXXX_PREFIX/Library/Audio/Plug-Ins/LV2" --prefix="$MIXXX_PREFIX" --no-plugins
```

Comment:
Shouldn't `--lv2dir` be `/Library/Audio/Plug-Ins/LV2` [according to the LV2 standard](http://lv2plug.in/pages/filesystem-hierarchy-standard.html)?

Reply:
If you don't set --lvd2dir then this installer script tries to install a bunch of manifest junk to /Library/Audio/Plug-Ins/LV2 on the build VM, so I changed it to $PREFIX//Library/Audio/Plug-Ins/LV2. I don't know what we're supposed to do with these manifests -- does Mixxx need to install them on the user's computer? We don't have an installer on macOS, the user just drops the Mixxx.app bundle into /Applications.

Reply:
> I think you need to write "closes #15", not just "#15" in the commit message for GitHub to automatically close the issue when this is merged.

Yea, I didn't because #15 is about Mac and Windows support so this PR doesn't fully fix it.

Reply:
Put another way, we don't want to change the build VM's global state since each environment should be hermetic, so I had this write to `$MIXXX_PREFIX/Library/Audio/Plug-Ins/LV2` instead of `/Library/Audio/Plug-Ins/LV2`. 

From what I can tell, this option only affects where files are written -- I grepped for `$MIXXX_PREFIX` in all of the generated files and it doesn't occur.

Reply:
Oh, I presumed `--lv2dir` set the default plugin search path, but it seems I was wrong. It's not clear to me what these manifests do or if they're really necessary. The README in schemas.lv2 says "These are included with LV2 and installed as a bundle to support validation and
more intelligent use of data by hosts."

Reply:
Alright, well I'll merge this now. If there is an issue with macOS builds detecting LV2 plugins, let's keep this in mind.